### PR TITLE
Small cleanup

### PR DESCRIPTION
**Pull request by shadchin**

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -26,38 +26,29 @@
 
 [tox]
 envlist =
-  py26,py27,py32,py33,py34,
-  py27-monolithic,py33-monolithic,py34-monolithic,
-  py27-not-relaxed-strides,py33-not-relaxed-strides,py34-not-relaxed-strides
+  py27,py34,py35,py36,
+  py27-not-relaxed-strides,py34-not-relaxed-strides,py35-not-relaxed-strides,py36-not-relaxed-strides
```
Comment (charris) [inline tox.ini]: Why drop the monolithic (whatever they are) builds? Not sure we need all the `not-relaxed-strides` testing either, as `relaxed-strides` is now the default,  but in any case that line should be broken.

@njsmith @pv Thoughts?

**Comment by charris** [comment]
Why did you remove `numpy/core/include/numpy/_numpyconfig.h.in`? You need to describe what you have done and why in the description section of the PR.

File: tox.ini (hunk 1 above)
Comment (shadchin) [inline tox.ini]: That's my fault. I would like to delete unused NPY_SEPARATE_COMPILATION

Comment (shadchin) [inline tox.ini]: Err, if remove NPY_SEPARATE_COMPILATION, then pyXX-monolithic == pyXX, therefore pyXX-monolithic unnecessary

**Comment by charris** [comment]
I'd break this into at least two PR's. The PR header is also non-informative and could use improvement.

**Comment by shadchin** [comment]
Break for BLD and TST? Should I close the PR?

**Comment by charris** [comment]
I checked, and that file was needed for the bento build, for which support was removed in numpy 1.11.0. That should be mentioned in the commit message for the file removal and also in the PR description.

So the need for `PYTHON_HAS_UNICODE_WIDE` also went away for Bento. I think a good PR title would be something like 
```
MAINT: Remove files and constants that were only needed for Bento.
```

I think the thing to do is separate the bento cleanup and tox changes into two PRs.

**Comment by charris** [comment]
Yes, go ahead and close this. Thanks for tracking down those unused bits.

**Comment by juliantaylor** [comment]
_numpyconfig.h is used to export the used matlib in a header. Though it is probably not really needed, only use I can think if is using it to ifdef on vectorized math functions but that can probably be done in other ways better.

**Comment by charris** [comment]
@juliantaylor The file removed is `_numpyconfig.h.in` which was only used by bscript.

**Comment by shadchin** [comment]
Thank you for help with description PR

**Comment by juliantaylor** [comment]
no it is also used in `numpy/distutils/misc_util.py`

**Comment by charris** [comment]
I don't see it.
```
charris@fc [numpy.git (master)]$ grep -r _numpyconfig\.h\.in numpy
numpy/core/setup.py:            join('include', 'numpy', '_numpyconfig.h.in'),

charris@fc [numpy.git (master)]$ grep -r _numpyconfig\.h\.in build
build/lib.linux-x86_64-2.7/numpy/core/setup.py:            join('include', 'numpy', '_numpyconfig.h.in'),

```

**Comment by juliantaylor** [comment]
the .h.in file gets transformed to .h, so you have to search for _numpyconfig.h

**Comment by charris** [comment]
The question is, where does it get transformed? I note that numpy builds with/without that file and the resulting `_numpyconfig.h` is the same in both cases.

**Comment by charris** [comment]
I think it was only needed for Bento, where it was transformed in `numpy/core/bscript`. If you checkout v1.10.0 you can find it in that script.

**Pull request closed without merging**